File: 박유정/9월/4주차/0920/12926.py
def solution(s, n):
    low = "abcdefghijklmnopqrstuvwxyz"
    up = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
    answer = ''
    for char in s:
        if char in low:
            index = low.find(char)+n
            answer += low[index%26]
        elif char in up:
            index = up.find(char)+n
            answer += up[index%26]
        else:
            answer += " "
    return answer

File: 구은성/11월/1109/12926.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        char chArr[] = s.toCharArray();
        
        for(char ch: chArr) {
            if(ch >= 'a' && ch <= 'z') {
                ch = (char)((ch - 'a' + n) % 26 + 'a');
            } else if (ch >= 'A' && ch <= 'Z') {
                ch = (char)((ch - 'A' + n) % 26 + 'A');
            }
            answer += ch;
        }
        
        return answer;
    }
}